infrastructure/database.tf

```diff
@@ -1,6 +1,9 @@
 data "aws_ssm_parameter" "db_password" { name = var.ssm_db_password }
 data "aws_ssm_parameter" "db_username" { name = var.ssm_db_username }
 
+data "aws_ssm_parameter" "pe_db_password" { name = var.ssm_pe_db_password }
+data "aws_ssm_parameter" "pe_db_username" { name = var.ssm_pe_db_username }
+
```

Comment:
Do we need to add another statement here for pe_db_name or should I create a function like there is at the very end of this file that sends the name of the db found on line 97

resource "aws_ssm_parameter" "crossfeed_send_db_name" {
  name      = var.ssm_db_name
  type      = "SecureString"
  value     = aws_db_instance.db.name
  overwrite = true
  tags = {
    Project = var.project
  }
}

Reply:
I don't think we need the function, I just wanted to make sure it wasn't necessary

Reply:
I don't think we need it. The reason we do that code for crossfeed_send_db_name is because the main crossfeed db is automatically created by RDS, so we use the output of the rds resource to get that value and then set it in SSM.

On the other hand, for the P&E database, we set that value ourselves and save it in SSM, so the current code should work.